docs/core/development/policy/release-process.rst

```diff
@@ -199,9 +198,9 @@ Cut the tarballs & installers
 
    - ``python setup.py sdist --formats=bztar -d /tmp/twisted-release``
 
-2. Build Windows MSI
+2. Build Windows whl
```

Comment:
I assume a `whl` is a "wheel".  Maybe call it that here and leave the file extension in the detail below?